front/src/views/BoxCreate/BoxCreateView.tsx

```diff
@@ -222,27 +240,36 @@ function BoxCreateView() {
   };
 
   // Handle Loading State
-  if (qrCodeExists.loading || allFormOptions.loading || createBoxMutationState.loading) {
+  if (qrCodeExists.loading || allFormOptions.loading || createBoxMutationState.loading)
     return <APILoadingIndicator />;
-  }
 
   // TODO: handle errors not with empty div, but forward or roll data back in the view
   if (
     !qrCodeExists.data?.qrExists ||
     qrCodeExists.error ||
     allLocations === undefined ||
     allProducts === undefined
-  ) {
+  )
     return <div />;
```

Comment:
add an alert here with the message: "Could not fetch QR, Location and Product data! Please try reloading the page."